dbfit-java/core/src/main/java/dbfit/fixture/ExecuteProcedureExpectException.java

```diff
@@ -0,0 +1,38 @@
+package dbfit.fixture;
+
+import dbfit.api.DBEnvironment;
+import dbfit.api.DbEnvironmentFactory;
+import dbfit.util.ExpectedBehaviour;
+
+public class ExecuteProcedureExpectException extends ExecuteProcedure {
+    private boolean excNumberDefined = false;
+    private int excNumberExpected;
+
+    public ExecuteProcedureExpectException() {
+        this.environment = DbEnvironmentFactory.getDefaultEnvironment();
+    }
+
+    public ExecuteProcedureExpectException(DBEnvironment dbEnvironment, String procName,
+                            int expectedErrorCode) {
+        this.procName = procName;
+        this.environment = dbEnvironment;
+        this.excNumberDefined = true;
+        this.excNumberExpected = expectedErrorCode;
+    }
+
+    public ExecuteProcedureExpectException(DBEnvironment dbEnvironment, String procName) {
+        this.procName = procName;
+        this.environment = dbEnvironment;
+        this.excNumberDefined = false;
+    }
+
+    @Override
+    protected ExpectedBehaviour getExpectedBehaviour() {
+        return excNumberDefined ? ExpectedBehaviour.SPECIFIC_EXCEPTION : ExpectedBehaviour.ANY_EXCEPTION;
```

Comment:
Yes, it looks good now. What do you mean by min number of references? (In general I think we have a Checkstyle will raise an error if we have unused imports).